protected/views/layouts/new_datasetpage.php

```diff
@@ -6,6 +6,8 @@
     <meta name="language" content="en" />
     <meta name="robots" content="noindex">
     <meta name="googlebot" content="noindex">
+    <meta name="citation_doi" content="<?php echo $mainSection->getReleaseDetails()['full_doi']; ?>">
+
```

Comment:
Hi @pli888 

I tried to added `full_doi` to here, which will make it appear inside `<head>`, but `$mainSection` becomes an undefined variable when calling it.

Reply:
Hi @kencho51, @pli888 

You can use the same approach as for the title for the HTML Page which specifified in \<head\>, therefore in the layout, but is specific to the dataset. (Look for ``pageTitle`` at the top of ``protected/views/dataset/view.php`` )